cqrs/eventbus/kafka/subscriber.go

```diff
@@ -2,6 +2,7 @@ package kafka
 
 import (
```

Comment:
Please move the `cqrs` package out of the `kit` repo #9

Reply:
OK. It will be done in near future ;D